### PR TITLE
Adding Plugin Lenovo code

### PR DESCRIPTION
**Pull request by amar-shalgar**

### CODE REVIEW

**Review by skhan-anjum — APPROVED** [review]

File: .github/workflows/build_deploy_test.yml [hunk 1]
```diff
@@ -45,14 +45,14 @@ jobs:
         export HOSTIP=${HOSTIP}
         make all
         sleep 15s
-        b=`pgrep -f -c 'svc-api|svc-account-session|svc-aggregation|svc-events|svc-systems|svc-task|svc-fabrics|svc-telemetry|svc-managers|svc-update|plugin-redfish|urp|plugin-dell'`
+        b=`pgrep -f -c 'svc-api|svc-account-session|svc-aggregation|svc-events|svc-systems|svc-task|svc-fabrics|svc-managers|svc-update|plugin-redfish|urp|plugin-dell|plugin-lenovo'`
```
Comment (bharath-b-hpe) [inline .github/workflows/build_deploy_test.yml:48]: svc-telemetry has got removed

File: odim-controller/helmcharts/lenovoplugin/lenovoplugin-config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+odimra:
```
Comment (bharath-b-hpe) [inline odim-controller/helmcharts/lenovoplugin/lenovoplugin-config.yaml:1]: Please remove parameters in odimra group

File: .github/workflows/build_deploy_test.yml (hunk 1 above)
Comment (amar-shalgar) [reply .github/workflows/build_deploy_test.yml]: Thank you Bharath, corrected it

File: odim-controller/helmcharts/lenovoplugin/lenovoplugin-config.yaml (hunk 2 above)
Comment (amar-shalgar) [reply odim-controller/helmcharts/lenovoplugin/lenovoplugin-config.yaml]: Removed the odimra group

**Review by jeevan-kamkar — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Pull request merged**